app/src/main/java/com/example/fundatecheroes/login/view/LoginActivity.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.fundatecheroes.login.view
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.example.fundatecheroes.R
+import com.example.fundatecheroes.databinding.ActivityLoginBinding
+import com.example.fundatecheroes.home.view.HomeActivity
+import com.example.fundatecheroes.profile.view.ProfileActivity
+
+class LoginActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityLoginBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+
+        val view = binding.root
+
+        setContentView(view)
```

Comment:
Como não vamos utilizar a variavel view, podemos usar o binding.root diretamente
```suggestion
        setContentView(binding.root)
```